README.md

```diff
@@ -2,32 +2,45 @@
 
 An OpenTelemetry backend in a Docker image.
 
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable-next-line MD013 -->
```

Comment:
why do we need annotations for prettier, markdownlint and editorconfig-checker now?

Reply:
I added the markdownlint one to remove the squiggles I get in VS Code, but then prettier complains that there isn't a new line between the `markdownlint-disable-next-line`, but adding that would make the `next-line` not work, so it needs disabling around that block.

If there's a neater way to make all three happy for this one line I can change it (or I can just make the alt text short enough to not go over 120).

Reply:
maybe we can let only one linter check the line length by telling the others to ignore the rule

Reply:
I've added a `.prettierignore` to ignore Markdown sotwo of the extra lines can be removed. For the sake of just the one inline suppression I reckon that's good enough.